handlers/diagnostics.py

```
@@ -1,432 +1,432 @@
+
+def _check_ollama(base_url: str) -> dict[str, Any]:
+    """Check if Ollama is reachable and list loaded models.
+
+    Sync helper — call from async handlers via ``asyncio.to_thread`` or use
+    ``check_ollama_async`` to avoid blocking the event loop.
+    """
+    try:
+        r = httpx.get(f"{base_url}/api/tags", timeout=5.0)
+        models = [m["name"] for m in r.json().get("models", [])]
+        return {"reachable": True, "model_count": len(models), "models": models}
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)